core/pom.xml

```diff
@@ -487,9 +487,9 @@ THE SOFTWARE.
       <version>20110809</version>
     </dependency>
     <dependency>
-      <groupId>commons-collections</groupId>
-      <artifactId>commons-collections</artifactId>
-      <version>3.2.1</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
```

Comment:
This is a breaking change that will break plugins that inherit and use this dependency from core.